msi/mkrelease.ps1

```diff
@@ -0,0 +1,120 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    [String] $JenkinsVersion,
+    [string] $MSBuildPath = '',
+    [bool] $UseTracing = $false,
+    [String] $ProductName = '',
+    [String] $ProductSummary = '',
+    [String] $ProductVendor = '',
+    [String] $ArtifactName = '',
+    [String] $BannerBmp = '',
+    [String] $DialogBmp = '',
+    [String] $InstallerIco = ''
+)
+
+if($UseTracing) { Set-PSDebug -Trace 1 }
+
+$ErrorActionPreference = "Stop"
+
+Function Get-Jenkins([String] $JenkinsVersion, [String] $outputPath='.') {
+    $isLts = $JenkinsVersion.Split('.').Length -gt 2
+
+    $warUrl = "http://mirrors.jenkins.io/war/${JenkinsVersion}/jenkins.war"
+    $warSha256Url = "http://mirrors.jenkins.io/war/${JenkinsVersion}/jenkins.war.sha256"
+
+    if($isLts) {
+        $warUrl = "http://mirrors.jenkins.io/war-stable/${JenkinsVersion}/jenkins.war"
```

Comment:
Wouldn't it be better to paramatrize this warUrl in order to be able to download the jenkins.war from different maven repository like [here](https://github.com/olblak/packaging/blob/25a3ed58d04803ed772e5bbdf76e4660fca6a78e/utils/getJenkinsVersion.py#L50) the benefits of this would be able to package a 'private' version either from repo.jenkins-ci.org or from a temporary package manager

Reply:
Sure, that would be easy to do. I'll update the PR.

Reply:
It looks like some repos have a sha hash and some don't. Are you doing any verification of the hashes in your stuff for the deb/rpm/etc?

Reply:
I removed the SHA hash checking as it is not being done for the other package types.